cmd/zoekt-sourcegraph-indexserver/main.go

```diff
@@ -255,7 +260,6 @@ func (s *Server) loggedRun(tr trace.Trace, cmd *exec.Cmd) (err error) {
 			}
 
 			tr.LazyPrintf("success")
-			debug.Printf("ran successfully %s", cmd.Args)
```

Comment:
How will we know when the command is done without this? I suppose that is fine in practice?

Reply:
We have a separate log message if the index was updated "updated index ..." which tells us that indexing was successful. 

If other logged commands, such as `git config` fail, `gitIndex` returns with an error and we log that, too.